src/environments.js

```diff
@@ -10,27 +10,19 @@ import type {ArgType, Mode, StyleStr} from "./types";
  *  - mode: current parsing mode.
  *  - envName: the name of the environment, one of the listed names.
  *  - parser: the parser object.
- *  - positions: the positions associated with these arguments from args.
  */
 type EnvContext = {
     mode: Mode,
     envName: string,
     parser: Parser,
-    positions: number[],
 };
 
 /**
- * List of ParseNodes followed by an array of positions.
- * Returned by `Parser`'s `parseArguments`.
- */
-type ParseResult = (ParseNode | number[])[] | ParseNode;
```

Comment:
FYI: I'd confused myself in the last PR where I used this as the type: I'd confused the local `parseArray` method with `Parser`'s `parseArguments` method. Sorry about that.